packages/compass-aggregations/src/components/create-view-modal/create-view-modal.tsx

```diff
@@ -10,9 +10,12 @@ import {
   TextInput,
 } from '@mongodb-js/compass-components';
 import { createView, changeViewName, close } from '../../modules/create-view';
-import type { LoggerAndTelemetry } from '@mongodb-js/compass-logging/provider';
-import { withLoggerAndTelemetry } from '@mongodb-js/compass-logging/provider';
+import type { Logger } from '@mongodb-js/compass-logging/provider';
+import { withLogger } from '@mongodb-js/compass-logging/provider';
 import type { CreateViewRootState } from '../../stores/create-view';
+import { createTrack } from '@mongodb-js/compass-telemetry';
+
+const track = createTrack();
```

Comment:
This skips service injection, doesn't it?

Reply:
updating